lib/utils.pm

```diff
@@ -190,6 +191,16 @@ sub is_casp {
     return check_var('FLAVOR', $flavor);
 }
 
+# Checks if hypervisor is xen with optional type [pv|hvm] parameter
+sub is_xen {
+    my $type = shift;
+    return 0 unless check_var('VIRSH_VMM_FAMILY', 'xen');
+    return 1 unless $type;
+
+    return check_var('VIRSH_VMM_TYPE', 'linux') if $type eq 'pv';
+    return check_var('VIRSH_VMM_TYPE', $type);
+}
```

Comment:
I think we need to have single variable for XEN=PV otherwise it create a lot of confusion. Don't care which one exactly but single one